package.json

```diff
@@ -35,26 +35,9 @@
   },
   "pnpm": {
     "overrides": {
-      "cross-spawn": "7.0.6",
-      "vue": "3.5.19",
       "@nestjs-modules/mailer>mjml": "5.0.0-alpha.4",
-      "subscriptions-transport-ws>ws": "7.5.10",
-      "braces": "3.0.3",
-      "send": "0.19.0",
-      "pug": "3.0.3",
-      "body-parser": "1.20.3",
-      "path-to-regexp@3.2.0": "3.3.0",
-      "path-to-regexp@0.1.10": "0.1.12",
-      "micromatch@<4.0.8": "4.0.8",
-      "dset@3.1.3": "3.1.4",
-      "nanoid@3.3.7": "3.3.8",
       "execa@0.10.0": "2.0.0",
-      "@babel/runtime@<7.26.10": "7.26.10",
-      "apiconnect-wsdl": "2.0.36",
-      "@xmldom/xmldom": "0.8.10",
-      "brace-expansion@2.0.1": "2.0.2",
-      "brace-expansion@1.1.11": "1.1.12",
-      "form-data@4.0.2": "4.0.4"
+      "sha.js@2.4.11": "2.4.12"
     },
```

Comment:
As discussed, please review the dependency overrides and verify their impact on version resolution by checking the dependency tree.

Reply:
Pushed the changes, @jamesgeorge007, PTAL